js/ux-capture.js

```diff
@@ -22,91 +33,114 @@ window.UX = (function() {
   let onMark;
   let onMeasure;
 
-  return {
-    expect: function(zones) {
-      expectedZones = zones.map(zone => {
-        // only create promises if zone contains any marks, otherwise just ignore it
-        if (zone.marks && zone.marks.length > 0) {
-          // wrap each string in the array into an object so we can attach record() methods to them
-          zone.marks = zone.marks.map(label => ({ label }));
-
-          // create a promise for each expected mark
-          const promises = zone.marks.map(
-            mark =>
-              new Promise((resolve, reject) => {
-                mark.record = () => {
-                  if (isUserTimingSupported) {
-                    // record the mark using W3C User Timing API
-                    performance.mark(mark.label);
-
-                    // if callback is specified, call it with mark label
-                    if (onMark) {
-                      onMark(mark.label);
-                    }
-                  }
-
-                  /**
-                   * Report same mark on Chrome/Firefox timeline
-                   *
-                   * keep in mind, these timestams are counted from timeline recording start
-                   * while UserTiming marks are counted from navigationStart event
-                   * however visually, they all will be offset by the same amount of time and align vertically on the charts
-                   *
-                   * (we'd provide a helper to highlight discrepancy, but unfortunately,
-                   * there is no way to know when in timeline did navigationStart event occured)
-                   */
-                  if (isConsoleTimeStampSupported) {
-                    console.timeStamp(mark.label);
+  window.UX.expect = function(zones) {
+    expectedZones = zones.map(zone => {
+      // only create promises if zone contains any marks, otherwise just ignore it
+      if (zone.marks && zone.marks.length > 0) {
+        // wrap each string in the array into an object so we can attach record() methods to them
+        zone.marks = zone.marks.map(label => ({ label }));
+
+        // create a promise for each expected mark
+        const promises = zone.marks.map(
+          mark =>
+            new Promise((resolve, reject) => {
+              mark.record = () => {
+                if (isUserTimingSupported) {
+                  // record the mark using W3C User Timing API
+                  window.performance.mark(mark.label);
+
+                  // if callback is specified, call it with mark label
+                  if (onMark) {
+                    onMark(mark.label);
                   }
+                }
+
+                /**
+                 * Report same mark on Chrome/Firefox timeline
+                 *
+                 * keep in mind, these timestams are counted from timeline recording start
```

Comment:
nitpick, typo on `timestams` should be `timestamps`

Reply:
Typos are the hardest to find, thank you ;)